### PR TITLE
Fix dependencies

### PR DESCRIPTION
**Pull request by samdark**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | -

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,13 +23,12 @@
         "cycle/annotated": "^2.0",
         "cycle/migrations": "^1.0",
         "hiqdev/composer-config-plugin": "^1.0@dev",
-        "symfony/console": "^4.3",
+        "yiisoft/yii-console": "^3.0@dev",
```
Comment (roxblnfk) [inline composer.json]: Do dependency sorting at once

File: composer.json [hunk 2]
```diff
@@ -19,17 +19,16 @@
...
         "hiqdev/composer-config-plugin": "^1.0@dev",
-        "symfony/console": "^4.3",
         "yiisoft/aliases": "@dev",
-        "yiisoft/friendly-exception": "@dev"
+        "yiisoft/friendly-exception": "@dev",
+        "yiisoft/yii-console": "^3.0@dev"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.2",
-        "phan/phan": "^2.2"
```
Comment (roxblnfk) [inline composer.json]: now we can remove this section?
```
    "scripts": {
        "phan": "phan --progress-bar -o analysis.txt"	
    },
```

Comment (samdark) [reply composer.json]: Kind of but I think we can get it back when it's fixed in phan repo.

Comment (roxblnfk) [reply composer.json]: https://github.com/phan/phan/pull/3559 has been merged

**Pull request merged**